.gitignore

```diff
@@ -1,6 +1,5 @@
 generated/
 node_modules/
-package-lock.json
```

Comment:
A caveat to being able to cache `node_modules` and have fast builds is checking in our lock file. I think it would be easy to set up an automated workflow to keep this up to date for us automatically.